### DISCUSSION TITLE
视频打开存在过曝问题

### DISCUSSION THREAD

**Discussion by SherFuKun** [ywmoyue/biliuwp-lite#232] [Q&A]
一直以来都存在此问题，本人使用的是surface go win10设备，在打开视频后出现视频过曝问题，每次只能手动拉底视频亮度（使用其他软件正常），不知如何解决。
![屏幕截图 2023-07-27 203612](https://github.com/ywmoyue/biliuwp-lite/assets/68261996/3eb06320-6a64-4030-8950-d86dfb76331b)

**Comment by ywmoyue** [comment]
我用surface go试了下，只发现色彩和其他设备屏幕不一致，但没看出过曝问题，我后面再用其他播放器试试

**Comment by Clement-WANGWH** [comment]
我是surface pro8，也有相同问题

**Comment by xiaoyaocz** [comment]
可以看看是不是跟这里面的设置有关
![微信截图_20230911095741](https://github.com/ywmoyue/biliuwp-lite/assets/16559082/98f14242-a9c0-4853-bec8-92b096637a79)
